lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -117,9 +117,15 @@
 )
 KERNEL_VERSION_PATTERN = re.compile(r"Linux version (?P<kernel_version>.+?) ", re.M)
 
+# 2021/03/31 00:05:17.431693 INFO Daemon Azure Linux Agent Version:2.2.38
+WALA_VERSION_PATTERN = re.compile(
+    r"Azure Linux Agent Version:(?P<wala_version>.+?)[\n\r]", re.M
```

Comment:
is wala some outdated name? can we stick to the current one, if so?

Reply:
Not sure what's current name, but it's used in database and not exposed to runbook. Let's keep as is.

Reply:
@simonxiaoss any clue?